platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/Compactable.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.merkledb;
+
+import edu.umd.cs.findbugs.annotations.Nullable;
+import java.io.IOException;
+import java.util.LongSummaryStatistics;
+import java.util.function.BiConsumer;
+
+/**
+ * This interface indicates that the data storage files in the implementing class can be compacted
+ */
+public interface Compactable {
```

Comment:
Nitpick: `Compactable` should be spelled `Compactible` (c.f. [Oxford English](https://www.oed.com/dictionary/compactible_adj)).

Reply:
Idea complains about both variants, and, surprisingly enough, `compactable` is more popular in Google search than `compactible`. However, point taken - I trust Oxford English more :)

Reply:
It's a very commonly misspelled word, unfortunately.